.ci.yaml

```diff
@@ -2549,6 +2549,24 @@ targets:
       - bin/**
       - .ci.yaml
 
+  - name: Mac hello_world_macos__compile
+    bringup: true # New target https://github.com/flutter/flutter/issues/109633
+    recipe: devicelab/devicelab_drone
+    timeout: 60
+    properties:
+      tags: >
+        ["devicelab", "hostonly"]
+      task_name: hello_world_macos__compile
+
+  - name: Mac_arm64 hello_world_macos__compile
```

Comment:
Sorry for the back and forth here. Host only targets on arm64 is now blocked on issues in https://github.com/flutter/flutter/pull/109889.
To unblock, you may want to run it in `Mac_arm64_ios` platform.

Reply:
https://github.com/flutter/flutter/pull/109891#discussion_r952032899

Reply:
https://github.com/flutter/flutter/pull/109891#discussion_r953065815